%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Trey Hunner (http://www.treyhunner.com/)
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass{resume} % Use the custom resume.cls style
\usepackage[dvipsnames]{xcolor}
\usepackage{hyperref}

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}
\renewcommand{\thefootnote}{\fnsymbol{footnote}}

\usepackage{enumitem}

\name{ARIELLA GLADSTEIN, PhD} % Your name
%\address{} % Your address
\address{algladstein@gmail.com} % Your phone number and email
% \address{\small \url{https://ariella-gladstein.netlify.app}  \\ \url{https://github.com/agladstein} \\ \url{www.linkedin.com/in/ariella-gladstein}} % Your phone number and email
\address{\small \url{https://github.com/agladstein} \\ \url{www.linkedin.com/in/ariella-gladstein}} % Your phone number and email
\renewenvironment{rSection}[1]{
\sectionskip
\textcolor{RoyalPurple}{\MakeUppercase{#1}}
\sectionlineskip
\hrule
\begin{list}{}{
\setlength{\leftmargin}{1.5em}
}
\item[]
}{
\end{list}
}

\begin{document}


\begin{center}
\textcolor{RoyalPurple}{\bf \em Accomplished computational biologist with deep expertise in population genetics, machine learning, and large-scale genomic data analysis. Proven ability to design, develop, and implement machine learning algorithms to analyze genomic datasets, collaborate with multidisciplinary teams, and communicate findings effectively.}
\end{center}

\textcolor{RoyalPurple}{\bf Expertise:}
\textcolor{RoyalPurple}{\em population genetics, computational genomics, machine learning, cloud computing, statistics}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Education}

{\bf University of Arizona, Tucson, AZ} \hfill {\em August 2018} \\ 
PhD in Ecology and Evolutionary Biology, Minor in Mathematics 

{\bf Beloit College, Beloit, WI} \hfill {\em May 2011} \\ 
B.S. in Mathematical Biology \& Russian, Cum Laude \\
Departmental Honors: Mathematical Biology

\end{rSection}

%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Experience}

% {\bf Open Source Contributor} \hfill  2024\\ 
% {\em Tskit community} \hfill {\em Remote}\\
% Contirbuted to writing documentation and tutorials for the population genomics tskit ecosytem.

{\bf Research Scientist, Computational Biology} \hfill  2021 - 2024\\ 
{\em Ancestry Science} \hfill {\em Embark Veterinary, Remote}\\
Developed dog breed ancestry and relative-matching products using AWS cloud infrastructure and state of the art computational genomics methods.
\begin{itemize}[noitemsep, topsep=0pt]
    \item Proposed and developed a machine learning approach for village dog classification, enhancing accuracy and scalability.
    \item Conducted literature reviews, tested, and reported recommendations on published methods, including advanced machine learning methods, for population clustering and relative inference.
    \item Collaborated with scientists and engineers to improve local ancestry inference, implementing algorithm improvements for uncertainty estimates.
    \item Played a key role in improving AWS pipeline automation from 40\% to over 95\% and scaling processes from hundreds of thousands to millions of samples.
    \item Collaborated with a cross-functional team on a science initiative projected to save \$7.5 million annually. 
    \item Served on the Diversity, Equity, Inclusion, and Belonging (DEIB) committee, co-authoring an Inclusive Norms document and contributing to various DEIB initiatives.
\end{itemize}
    
{\bf Postdoctoral Fellowship} \hfill  2018 - 2020\\ 
{\em Schrider Lab} \hfill {\em Department of Genetics, University of North Carolina, Chapel Hill}\\
Developed deep learning models for population genetics inference. Contributed to writing documentation and tutorials for the population genomics tskit software ecosystem. Actively participated in the PopSim Consortium, contributing to the development of the stdpopsim library.


{\bf Dissertation Research} \hfill  2011 - 2018\\ 
{\em Hammer Lab} \hfill {\em Ecology and Evolutionary Biology, University of Arizona}\\
Developed and applied computational methods for population genetics analysis, addressing ascertainment bias in identifying runs of homozygosity and demographic inference. Created and released software for high-throughput genomic simulations. Applied Approximate Bayesian Computation (ABC) to model the demographic history of Ashkenazi Jews.

\end{rSection}


%----------------------------------------------------------------------------------------
%	Skills
%----------------------------------------------------------------------------------------

\begin{rSection}{Skills}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{6ex}} l }
Programming & Python, R, Bash, unit testing, profiling, multiprocessing, Git\\
Machine Learning & Scikit-learn, Keras, TensorFlow\\
Data Science & Numpy, Pandas, SciPy, Matplotlib, Seaborn, Plotly \\
Cloud Computing & AWS, SageMaker, JupyterHub, Kubernetes, Linux/Unix \\
Genomics & PLINK, Bedtools, VCFtools, phasing, imputation, haplotype detection, population \\ & structure, genome simulation\\
Reproducibility & Workflow development (Pegasus, Makeflow, Snakemake), Docker, Jupyter, Knitr \\
Documentation & \LaTeX, Markdown, reStructuredText, JupyterBook, Confluence \\
\end{tabular}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Teaching Experience
%----------------------------------------------------------------------------------------

\begin{rSection}{Teaching Experience}

    {\bf Introduction to using stdpopsim Workshop} \hfill 2020, {\em Virtual, PopSim Consortium} \\ 
    {\em Designed and presented interactive workshop on using stdpopsim, the standard library for reproducible, bug-free simulations of genetic datasets from published demographic histories.}

    {\bf NSF Cyber Carpentry Workshop} \hfill 2019, {\em University of North Carolina, Chapel Hill} \\ 
    {\em Assisted with 2-week workshop, where participants learned best practices for data-intensive computing, cloud infrastructure, and deep learning through hands-on projects.}

    {\bf CyVerse Container Camp} \hfill 2018, {\em University of Arizona} \\ 
    {\em Assisted with 3-day hands-on workshop on using containerized workflows for more reproducible science.}

    {\bf Software Carpentry Workshop} \hfill 2017, {\em University of Arizona} \\ 
    {\em Assisted with Software Carpentry hands-on workshop on basic Unix/Bash, Python, and Git}

    {\bf ECOL 320 Genetics} \hfill 2014 - 2016, {\em University of Arizona} \\ 
    {\em Graduate Teaching Assistant}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Publications
%----------------------------------------------------------------------------------------

\begin{rSection}{Select Publications, Posters, \& Talks}

    \item \textbf{Gladstein A.L.}\footnote{Presenter}, et al. A Novel Machine Learning Method for Classification of Village Dogs.  Probabilistic Modeling in Genomics. 2023. Cold Spring Harbor, NY. (Poster)

    \item Baumdicker, F.*, Bisschop, G.*, Goldstein, D.*, Gower, G.*, Ragsdale, A. P.*, Tsambos, G.*, Zhu, S.*, $\ldots$, \textbf{Gladstein, A. L.}, $\ldots$, Kelleher, J. 2022. Efficient ancestry and mutation simulation with msprime 1.0. \textit{Genetics}. Volume 220, Issue 3. \url{https://doi.org/10.1093/genetics/iyab229}

    \item Adrion, J. R.*, Cole, C. B.*, Dukler, N.*, Galloway, J. G.*, \textbf{Gladstein, A. L.}\footnote{First author}, Gower, G.*, Kyriazis, C.C.*, Ragsdale, A.P.*, Tsambos, G.*, $\ldots$, Gravel, S., Gutenkunst, R.N., Lohmeuller, K.E., Ralph, P.L., Schrider, D.R., Siepel, A., Kelleher, J., Kern, A.D. 2020. A community-maintained standard library of population genetic models. \textit{eLife}. 9:e54967. doi: \url{https://doi.org/10.1101/2019.12.20.885129}
        
    \item \textbf{Gladstein A.L.}\textsuperscript{\textdagger}, Schrider R.D.  Demographic model selection with deep learning.  Probabilistic Modeling in Genomics. 2019. Aussois, France. (Poster)

    \item \textbf{Gladstein A.L.}* and Hammer M.F. 2019. Substructured population growth in the Ashkenazi Jews inferred with Approximate Bayesian Computation. \textit{Molecular Biology and Evolution}. 36(6): 1162-1171. doi: \url{https://dx.doi.org/10.1093/molbev/msz047}
        
    \item \textbf{Gladstein A.L.}* et al. 2018. SimPrily: A Python framework to simplify high-throughput genomic simulations. \textit{SoftwareX}, 7, 335-340. \url{https://doi.org/10.1016/j.softx.2018.09.003}
                    
    \item \textbf{Gladstein A.L.}\textsuperscript{\textdagger}, et al. The effect of SNP array ascertainment bias on the distribution of runs of homozygosity lengths. Annual Meeting of the American Society for Human Genetics. 2015. Baltimore, MD. (Poster)

\end{rSection}

%----------------------------------------------------------------------------------------
%	Awards, Scholarships, & Grants
%----------------------------------------------------------------------------------------

\begin{rSection}{Awards}

NSF XSEDE compute allocations (1e6 CPU hrs, 6,500 GPU hrs, 5.5 Tb storage) \hfill{\em 2017 - 2020}\\
NIH Computational and Mathematical Modeling of Biological Systems Traineeship (\$71,064) \hfill {\em 2013-14}\\
NSF Integrative Graduate Education and Research Traineeship in Genomics (\$97,083) \hfill {\em 2011-13}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Project Management
%----------------------------------------------------------------------------------------

\begin{rSection}{Project Management}

    Managed team of 5 interns, including computer scientists, software engineer, and mathematician, \hfill {\em 2017}\\ 
    working on code development and high throughput computing for bioinformatics.
    
\end{rSection}

%----------------------------------------------------------------------------------------
%	Other Activities
%----------------------------------------------------------------------------------------

\begin{rSection}{Other Activities}

{\bf Circus Arts} \hfill 2012 - present \\ 
{\em Aerial silks, aerial rope, lyra, flying trapeze, tightwire, handbalancing, contortion} 

{\bf Figure Skating} \hfill 1998 - 2013, 2021 - present \\ 
{\em Singles freestyle} 

\end{rSection}
%----------------------------------------------------------------------------------------


\end{document}