%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Trey Hunner (http://www.treyhunner.com/)
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass{resume} % Use the custom resume.cls style
\usepackage[dvipsnames]{xcolor}
\usepackage{hyperref}

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}
\renewcommand{\thefootnote}{\fnsymbol{footnote}}

\name{ARIELLA GLADSTEIN, PhD} % Your name
% \name{MY NAME, PhD}
%\address{} % Your address
\address{algladstein@gmail.com} % Your phone number and email
% \address{myemail@gmail.com}
\address{\small \url{https://ariella-gladstein.netlify.app}  \\ \url{https://github.com/agladstein} \\ \url{www.linkedin.com/in/ariella-gladstein}} % Your phone number and email
\renewenvironment{rSection}[1]{
\sectionskip
\textcolor{RoyalPurple}{\MakeUppercase{#1}}
\sectionlineskip
\hrule
\begin{list}{}{
\setlength{\leftmargin}{1.5em}
}
\item[]
}{
\end{list}
}

\begin{document}

\textcolor{RoyalPurple}{\bf Expertise:}
\textcolor{RoyalPurple}{\em population genetics, genomics, computational biology, data science, machine learning}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------


\begin{rSection}{Education}

{\bf University of Arizona, Tucson, AZ} \hfill {\em August 2018} \\ 
PhD in Ecology and Evolutionary Biology \\
Minor in Mathematics 

{\bf Beloit College, Beloit, WI} \hfill {\em May 2011} \\ 
B.S. in Mathematical Biology \& Russian, Cum Laude \\
Departmental Honors: Mathematical Biology

\end{rSection}

%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Experience}

% {\bf Open Source Contributor} \hfill  2024\\ 
% {\em Tskit community} \hfill {\em Remote}\\
% Contirbuted to writing documentation and tutorials for the population genomics tskit ecosytem.

{\bf Research Scientist, Computational Biology} \hfill  2021 - 2024\\ 
{\em Ancestry team} \hfill {\em Embark Veterinary, Remote}\\
Developed dog breed ancestry and relative-matching products, enhancing computational efficiency while maintaining rigorous quality standards. Played a key role in improving pipeline automation from 40\% to over 95\% within one year and scaling processes from hundreds of thousands to millions of samples. Collaborated with a cross-functional team on a science initiative projected to save \$7.5 million annually.
Served on the Diversity, Equity, Inclusion, and Belonging (DEIB) committee, co-authoring an Inclusive Norms document, designing and analyzing company-wide DEIB surveys, and contributing to various DEIB initiatives.


{\bf Postdoctoral Fellowship} \hfill  2018 - 2020\\ 
{\em Schrider Lab} \hfill {\em Department of Genetics, University of North Carolina, Chapel Hill}\\
Developed deep learning models for population genetics inference, utilizing high-throughput computing for simulations and high-performance GPUs, as well as a Kubernetes cluster, for model training. Contributed to writing documentation and tutorials for the population genomics tskit software ecosystem. Actively participated in the PopSim Consortium, contributing to the development of the stdpopsim library.

{\bf Dissertation Research} \hfill  2011 - 2018\\ 
{\em Hammer Lab} \hfill {\em Ecology and Evolutionary Biology, University of Arizona}\\
Developed and applied advanced computational methods in population genetics. Refactored a Python simulation pipeline to optimize memory usage and enable efficient parallel processing. Leveraged over 11 million CPU hours in under six months to run large-scale simulations on the Open Science Grid and multiple university HPCs, utilizing a high-throughput workflow.






\end{rSection}


%----------------------------------------------------------------------------------------
%	Skills
%----------------------------------------------------------------------------------------

\begin{rSection}{Skills}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{6ex}} l }
Software Development & Python, R, Bash, unit testing, profiling, multiprocessing, Git\\
HPC/HTC and Cloud & Linux/Unix, SLURM, PBS, AWS, JupyterHub, Kubernetes  \\
Reproducibility & Workflow development (Pegasus, Makeflow, Snakemake), containers \\ & (Singularity, Docker), virtual environments, Jupyter, Knitr \\
Genomics & PLINK, Bedtools, VCFtools, phasing, imputation, haplotype detection, population \\ & structure, genome simulation\\
Data Science & Numpy, Pandas, Scikit-learn, SciPy, TensorFlow, Keras, Matplotlib, Seaborn, Plotly \\
Documentation & \LaTeX, Markdown, reStructuredText, JupyterBook, Confluence \\
\end{tabular}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Teaching Experience
%----------------------------------------------------------------------------------------

\begin{rSection}{Teaching Experience}

    {\bf Introduction to using stdpopsim Workshop} \hfill 2020 \\ 
    {\em Designed and presented workshop} \hfill {\em Virtual, PopSim Consortium}

    {\bf Cyber Carpentry Workshop} \hfill 2019 \\ 
    {\em Volunteer Teaching Assistant} \hfill {\em University of North Carolina, Chapel Hill}

    {\bf Research Bazaar Workshop on R} \hfill 2018 \\ 
    {\em Volunteer Teaching Assistant} \hfill {\em University of Arizona}

    {\bf CyVerse Container Camp} \hfill 2018 \\ 
    {\em Volunteer Teaching Assistant} \hfill {\em University of Arizona}

    {\bf Software Carpentry on Unix/Bash, Python, and Git} \hfill 2017 \\ 
    {\em Volunteer Teaching Assistant} \hfill {\em University of Arizona}

    {\bf Tucson Womenâ€™s Hackathon workshop on Git} \hfill 2017 \\ 
    {\em Volunteer Teaching Assistant} \hfill {\em University of Arizona}

    {\bf ECOL 320 Genetics} \hfill Fall 2014, 2015, 2016 \\ 
    {\em Graduate Teaching Assistant} \hfill {\em University of Arizona}

    {\bf ECOL 182L Intro to Ecology and Evolutionary Biology Lab} \hfill Spring 2015 \\ 
    {\em Graduate Teaching Assistant} \hfill {\em University of Arizona}

    {\bf BIOL 247 Biometrics} \hfill Spring 2011 \\ 
    {\em Teaching Assistant} \hfill {\em Beloit College}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Awards, Scholarships, & Grants
%----------------------------------------------------------------------------------------

\begin{rSection}{Awards}

XSEDE Research allocation (250,000 CPU hrs) \hfill{\em 2020}\\
XSEDE Supplemental allocation (250,000 CPU hrs) \hfill{\em 2018, 2020}\\
XSEDE Startup allocation (100,000 CPU hrs, 6,500 GPU hrs, 5.5 Tb storage) \hfill{\em 2019}\\
XSEDE Startup allocation (150,000 CPU hrs) \hfill{\em 2017}\\
Open Science Grid User School (travel, lodging, meals, cost of program) \hfill{\em 2017}\\
NIH Computational and Mathematical Modeling of Biological Systems Traineeship (\$71,064) \hfill {\em 2013-14}\\
NSF Integrative Graduate Education and Research Traineeship in Genomics (\$97,083) \hfill {\em 2011-13}\\

\end{rSection}

%----------------------------------------------------------------------------------------
%	Workshops
%----------------------------------------------------------------------------------------


\begin{rSection}{Workshops and Hackathons}

SMBE satellite meeting on Speciation Genomics, Tjarno, Sweden (3 days) \hfill{\em 06/2019}\\
NCBI RNA-Seq in the Cloud hackathon, Chapel Hill, NC (3 days) \hfill {\em 03/2019}\\
Cyber Carpentry, Chapel Hill, NC (2 weeks) \hfill {\em 06/2018}\\
XSEDE HPC Workshop: Big Data, Tucson, AZ (2 days) \hfill {\em 02/2018}\\
Open Science Grid User School, Madison, WI (1 week) \hfill {\em 07/2017}

\end{rSection}


%----------------------------------------------------------------------------------------
%	Publications
%----------------------------------------------------------------------------------------

\begin{rSection}{Select Publications, Posters, \& Talks}

    \item \textbf{Gladstein A.L.}\footnote{Presenter}, et al. A Novel Machine Learning Method for Classification of Village Dogs.  Probabilistic Modeling in Genomics. 2023. Cold Spring Harbor, NY. (Poster)
        
    \item Baumdicker, F.*, Bisschop, G.*, Goldstein, D.*, Gower, G.*, Ragsdale, A. P.*, Tsambos, G.*, Zhu, S.*, $\ldots$, \textbf{Gladstein, A. L.}, et al. 2022. Efficient ancestry and mutation simulation with msprime 1.0. \textit{Genetics}. Volume 220, Issue 3. \url{https://doi.org/10.1093/genetics/iyab229}
    
    \item Adrion, J. R.*, Cole, C. B.*, Dukler, N.*, Galloway, J. G.*, \textbf{Gladstein, A. L.}\footnote{First author}, Gower, G.*, Kyriazis, C.C.*, Ragsdale, A.P.*, Tsambos, G.*, et al. 2020. A community-maintained standard library of population genetic models. \textit{eLife}. 9:e54967. doi: \url{https://doi.org/10.1101/2019.12.20.885129}
    
    
    \item \textbf{Gladstein A.L.}\textsuperscript{\textdagger}, Schrider R.D.  Demographic model selection with deep learning.  Probabilistic Modeling in Genomics. 2019. Aussois, France. (Poster)
    
    
    \item \textbf{Gladstein A.L.}* and Hammer M.F. 2019. Substructured population growth in the Ashkenazi Jews inferred with Approximate Bayesian Computation. \textit{Molecular Biology and Evolution}. 36(6): 1162-1171. doi: \url{https://dx.doi.org/10.1093/molbev/msz047}
    
    \item Inference of evolutionary history with Approximate Bayesian Computation. Open Science Grid All-Hands Meeting. 2018. Salt Lake City, Utah. (Talk)
    
    \item Code optimization for research scientists. Research Bazaar. 2018. Tucson, AZ. (Talk)
    
    \item \textbf{Gladstein A.L.}* et al. 2018. SimPrily: A Python framework to simplify high-throughput genomic simulations. \textit{SoftwareX}, 7, 335-340. \url{https://doi.org/10.1016/j.softx.2018.09.003}
    
    
    \end{rSection}

%----------------------------------------------------------------------------------------
%	Other Activities
%----------------------------------------------------------------------------------------

\begin{rSection}{Other Activities}

{\bf Circus Arts} \hfill 2012 - present \\ 
{\em Aerial silks, aerial rope, lyra, flying trapeze, tightwire, handbalancing, contortion} 

{\bf Figure Skating} \hfill 1998 - 2013 \\ 
{\em Singles freestyle} 

\end{rSection}
%----------------------------------------------------------------------------------------


\end{document}